index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Swadeshi</title>
+  </head>
+  <body>
+    <div class="root">
+      <div class="content">
+        <header class="header-container p-sm shadow-md">
+          <nav class="header__nav">
+            <div class="nav__logo-container">
+              <a href="./index.html" class="text-default"
+                ><i class="fas fa-store text-1"></i>&nbsp;
+                <h1 class="">
+                  Swadeshi
+                  <h4 class="bold-lg">.com</h4>
+                </h1>
```

Comment:
Dont use one heading tag inside another tag, Use span to do these kind of things